engine/src/flutter/impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -150,16 +152,6 @@ void ContextVK::Setup(Settings settings) {
 
   raster_message_loop_ = fml::ConcurrentMessageLoop::Create(
       ChooseThreadCountForWorkers(std::thread::hardware_concurrency()));
-  raster_message_loop_->PostTaskToAllWorkers([]() {
-    // Currently we only use the worker task pool for small parts of a frame
-    // workload, if this changes this setting may need to be adjusted.
-    fml::RequestAffinity(fml::CpuAffinity::kNotPerformance);
-#ifdef FML_OS_ANDROID
-    if (::setpriority(PRIO_PROCESS, gettid(), -5) != 0) {
-      VALIDATION_LOG << "Failed to set Workers task runner priority";
-    }
-#endif  // FML_OS_ANDROID
-  });
```

Comment:
Why did we get rid of this?

Reply:
Doesn't really do anything at this point, and was potentially adding more startup time since the worker tasks need to steal a core to activate for this task.